frontend/src/lib/auth/permissions.test.ts

```diff
@@ -310,9 +310,14 @@ DESTRUCTION_LIST_STATUSES.forEach((status) => {
       expect(canUpdateDestructionList(user, destructionList)).toBe(false);
     });
 
-    test("should not allow a user to update if they are not the assignee", () => {
+    test("should allow a user to update if they are not the assignee", () => {
```

Comment:
Shouldn't it be author instead of assignee?

Reply:
Better indeed, fixed